app/(auth)/lib/index.ts

```diff
@@ -1,7 +1,7 @@
 // export { getUserInitializationTask } from "./get-initialization-task";
 export { deleteOauthCredential } from "./delete-oauth-credential";
 export { getAuthCallbackUrl } from "./get-auth-callback-url";
-export { getUserSubscriptionId } from "./get-current-subscription";
+export { getUserTeamId } from "./get-current-subscription";
```

Comment:
nits:

I noticed that the file name `get-current-subscription` is outdated, so I was thinking it might be better to update it as well!

Reply:
@shige 
Thanks!
I renamed it, and found an almost identical function: `getCurrentTeam`.
So, I updated the code to use `getCurrentTeam`.